RELEASE.md

```diff
@@ -0,0 +1,50 @@
+Robot Web Tools Module Release TODOs
+====================================
+
+This document describes TODOs and checklists in order to release
+Robot Web Tool javascript modules([roslibjs](https://github.com/RobotWebTools/roslibjs), [ros2djs](https://github.com/RobotWebTools/ros2djs), [ros3djs](https://github.com/RobotWebTools/ros3djs)).
+
+### 0. Make sure that the releasing module is compatible with other RWT modules.
+
+use travis to check the compatibility?
+
+### 1. Generate CHANGELOG using [github-changes](https://github.com/lalitkapoor/github-changes). 
+
+```
+github-changes -o RobotWebTools -r roslibjs --only-pulls --use-commit-body -a -b develop
+```
+
+### 2. Bump a new version
+
+* Remove a SNAPSHOT postfix from package.json and bowers.json
+* Mark *upcoming* in CHAGELOG.md as the new release version
+* Tag the version
+
+### 3. Release modules
+
+**NPM**
+
+Keep all modules under @robot-web-tools scope? or in global scope? What are pros and cons?
+
+**Bowers**
```

Comment:
As far as I know bower is a thin wrapper on Git itself. Once a package is [registered](https://bower.io/docs/creating-packages/#register) and has a valid bower.json all you need to do to update is create and push a new git tag.

Reply:
Bower is kinda dying right now as the alternatives have won out over the years (sources [official1](https://bower.io/blog/2017/how-to-migrate-away-from-bower/), [official 2](https://github.com/bower/bower/blob/master/README.md#bower---a-package-manager-for-the-web), [reddit 3](https://www.reddit.com/r/javascript/comments/5ls30m/is_bower_dead_should_we_start_using_other/)). The current approaches people have been using is to just use NPM and not do much else special for frontend vs backend nodejs package management. I have very successfully used npm+[webpack](https://webpack.github.io/) or npm+[browserify](http://browserify.org/) with all of the RWT packages. [Yarn](https://yarnpkg.com/en/) is also a common alternative.

Reply:
As we are deprecating bower support, I will remove this section. Yarn can be added later.